moveit_commander/src/moveit_commander/move_group.py

```diff
@@ -571,7 +571,7 @@ def pick(self, object_name, grasp=[], plan_only=False):
     def place(self, object_name, location=None, plan_only=False):
         """Place the named object at a particular location in the environment or somewhere safe in the world if location is not provided"""
         result = False
-        if location is None:
+        if not location:
```

Comment:
This change makes this true with an empty list.  Is that what you wanted here?

Reply:
Yes, that was intended as I later need to check the type of objects inside the list. To do so I take the first element to check the type of objects inside the list (with the assumption that it will be a homogeneous list of objects of the same type). 
Therefore, it's safe to check for location[0] as if the list was empty it would have entered through this first if.